doc/changes/changes_16.1.3.md

```diff
@@ -0,0 +1,22 @@
+# Common module of Exasol Virtual Schemas Adapters 16.1.3, released 2022-09-??
+
+Code name: Fixed Geo Aggregation capabilities
+
+## Summary
+
+We deprecated the `FN_AGG_GEO_*` capabilities that were renamed in 7.1.alpha1. Please only use their new `FN_AGG_ST_*`
+counterparts.
+
+Additionally, we added an explanation of [dual-use functions and their capabilities](../development/api/capabilities_list.md#capabilities-for-dual-use-functions) to the developer guide.
```

Comment:
Update the link after the release on the release page, relative links won't work there.
Or use an absolute link here right now ;)